actors/common/src/main/java/org/sunbird/feed/FeedUtil.java

```diff
@@ -0,0 +1,61 @@
+package org.sunbird.feed;
+
+import java.util.*;
+import java.util.stream.IntStream;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.bean.ShadowUser;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.feed.impl.FeedFactory;
+import org.sunbird.models.user.Feed;
+
+/** this class will be used as a Util for inserting Feed in table */
+public class FeedUtil {
+  private static IFeedService feedService = FeedFactory.getInstance();
+  private static final String ORG_MIGRATION_ACTION = "OrgMigrationAction";
+  public static final String UNREAD = "unread";
+  public static final String READ = "read";
+
+  public static void saveFeed(ShadowUser shadowUser, List<String> userIds) {
+    Map<String, Object> reqMap = new HashMap<>();
+    reqMap.put(JsonKey.USER_ID, userIds.get(0));
```

Comment:
we are using only  first element from userIds then no need of taking list. 
either take only one userId or handle list properly